app/src/main/java/com/framgia/soundclound/screen/local/TrackLocalViewModel.java

```diff
@@ -0,0 +1,71 @@
+package com.framgia.soundclound.screen.local;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.ContentResolver;
```

Comment:
ERROR: Unused import: android.content.ContentResolver. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports